plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
Hmm.. not sure why you changed it this way.

It's still possible that people would have `fglrx_buggy_driver` because not everyone can update the drivers easily or quickly.

But since in this commit you don't check it anymore then whatever needs to know that it's buggy - doesn't know it anymore.

I'd think the correct way to fix it is that `fglrx_buggy_driver` would have a falsy value if the driver is good, and a truthy value if the driver is buggy.

This way the code would know if the driver is bad or not, instead of assuming it's good just because AMD released a good driver few days ago.